tests/wpt/mozilla/tests/mozilla/fetch_cannot_overwhelm_system.window.js

```diff
@@ -0,0 +1,15 @@
+// META: title=Ensure multiple fetch do not crash the browser.
+
+async_test(function(t) {
+  onload = t.step_func(function() {
+  	var step;
+  	var xhr
+  	var url = 'https://github.com/servo/servo/';
```

Comment:
I don't think it's a good idea to make requests to arbitrary URLs when running tests.

Reply:
Good point